collections/_release_4_5/entry-systems-input-sdl3-gamepad-input-driver.md

```diff
@@ -0,0 +1,27 @@
+---
+type: entry
+section: systems
+subsection: input
+rank: 0
+importance: 2
+anchor: sdl3-gamepad-input-driver
+title: SDL3 gamepad input driver
+blockquote: Gamepads galore!
+text: |
+  Gamepads are nowadays a given in modern PC gaming. Users expect their gamepad to just plug in and work. Not only that, but in order to deliver unique experiences, some gamepads are introducing new features.
+
+  Over time, issues accumulated in our gamepad driver implementation and missing features began to crop up. We were facing an ever-growing mountain.
+
+  That’s why we turned our heads to the SDL project. SDL is a well established and mature cross-platform library that handles audio, keyboard, mouse, joystick, and graphics. We determined that it’s now a net positive to defer the responsibility for gamepad handling to it.
```

Comment:
```suggestion
  That’s why we turned our heads to the [SDL project](https://wiki.libsdl.org/SDL3/FrontPage). SDL is a well established and mature cross-platform library that handles audio, keyboard, mouse, joystick, and graphics. We determined that it’s now a net positive to defer the responsibility for gamepad handling to it.
```
Add a link to SDL, since we're discussing it

Reply:
I agree we should link, but I think linking to the simpler https://libsdl.org/ URL would be better.